File: AuthHub.Models/Requests/ResetPasswordRequest.cs
﻿using System;

namespace AuthHub.Models.Requests
{
    public class ResetPasswordRequest
    {
        public int UserId { get; set; }
        public string Password { get; set; }
        public string ConfirmPassword { get; set; }
        public string VerificationCode { get; set; }
    }
}

File: authhub-ui/src/app/models/requests/ResetPasswordRequest.ts

export class ResetPasswordRequest{
    UserId: number | null | undefined;
    Password: string | null | undefined;
    ConfirmPassword: string | null | undefined;
    VerificationCode: string | null | undefined;

    constructor(init?: Partial<ResetPasswordRequest>){
        Object.assign(this, init)
    }
}